groups.go

```diff
@@ -33,19 +33,21 @@ type GroupsService struct {
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/groups.html
 type Group struct {
-	ID          int        `json:"id"`
-	Name        string     `json:"name"`
-	Path        string     `json:"path"`
-	Description string     `json:"description"`
-	Projects    *[]Project `json:"projects,omitempty"`
+	ID          int               `json:"id"`
+	Name        string            `json:"name"`
+	Path        string            `json:"path"`
+	Description string            `json:"description"`
+	Projects    *[]Project        `json:"projects,omitempty"`
+	Statistics  StorageStatistics `json:"statistics"`
```

Comment:
Please make this a pointer: `Statistics  *StorageStatistics`

And while you are editing this file, would you also want to fix the `Projects` line so it reads: 
`Projects []Project        'json:"projects"'`

Wonder how that one slipped through... Thx!